### ISSUE TITLE
Lab1  Exercise 8

### ISSUE THREAD

**Issue by BertLisser** [software-engineering-amsterdam/ST2017_WG_13#1]
The only thing  I found:
```
-- Those boys who are honest are the ones that have true accusations to those boys who are guilty
honest = [ x | x <- boys, y <- guilty, accuses x y]
dishonest = [x | x <- boys, y <- guilty, not (accuses x y)]
```
Better is:
```
-- Those boys who are honest are the ones that have true accusations to those boys who are guilty
honest =nub  [ x | x <- boys, y <- guilty, accuses x y]
dishonest =nub  [x | x <- boys, y <- guilty, not (accuses x y)]
```
Well done!

**Comment by olimoli9160** [comment]
Dear Bert,

This snippet appears to come from the implementation I made in my personal folder. Is this feedback for the personal repository of member 11784873 - Michael Olivari or for the team's submission? Because the team submission code is slightly different from the implementation I made.

Best,
Michael


On 15 Sep 2017, at 10:59, BertLisser <notifications@github.com<mailto:notifications@github.com>> wrote:


The only thing I found:

-- Those boys who are honest are the ones that have true accusations to those boys who are guilty
honest = [ x | x <- boys, y <- guilty, accuses x y]
dishonest = [x | x <- boys, y <- guilty, not (accuses x y)]


Better is:

-- Those boys who are honest are the ones that have true accusations to those boys who are guilty
honest =nub  [ x | x <- boys, y <- guilty, accuses x y]
dishonest =nub  [x | x <- boys, y <- guilty, not (accuses x y)]


Well done!

**Comment by BertLisser** [comment]
Dear Michael

I cannot find your submission. Where I have to look?

Bert 

     Bert

----- Original Message -----

**Comment by olimoli9160** [comment]
Dear Bert,

Our team's submission is in the Team directory -> folder 1 of the repository ST2017_WG_13.

Our personal repositories are in the Members directory, but those weren't intended to be graded. Of course, feedback on our personal work is always appreciated.

Best,

Michael


On 15 Sep 2017, at 11:56, BertLisser <notifications@github.com<mailto:notifications@github.com>> wrote:

Dear Michael

I cannot find your submission. Where I have to look?

Bert

Bert

----- Original Message -----

**Comment by BertLisser** [comment]
I found it. But where is the function "guilty"? I found the function "thief" with a different signature, however that was not the question.

----- Original Message -----

**Comment by AJGHM-Khonraad** [comment]
Hi Bert,

I can understand where your confusion is at. Maybe you could have a look at 

https://github.com/software-engineering-amsterdam/ST2017_WG_13/blob/master/Members/11424907-ToineKhonraad/1/Lab1_Exercise8.hs

We apparently had to agree as a team on the submission dynamics, and will do so for the next coming weeks. Sorry for the inconvenience.

Toine

**Comment by PaulSpencer** [comment]
this appears to be my fault.  I must have misread the question and not realized that one had to have exactly that signature, I thought it was about getting the correct result.
going forward, do we have to follow the exact signatures?

**Comment by BertLisser** [comment]
Yes. That is the first thing to what I glanced: are the types and names correct.
